### PR TITLE
Update person detection models

### PR DESCRIPTION
**Pull request by morkovka1337**

### CODE REVIEW

**Comment by morkovka1337** [comment]
run tests

File: external/mmdetection/configs/person-detection/person-detection-0301/template_experimental.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Description.
+model_template_id: Person_Detection_0301
+name: person-detection-0301
+task_type: DETECTION
+task_family: VISION
+instantiation: "CLASS"
+summary: Accurate person detection model (Resnet50-VFnet).
```
Comment (GalyaZalesskaya) [inline external/mmdetection/configs/person-detection/person-detection-0301/template_experimental.yaml]: I'd suggest specifying here up-down scenario, for covering which you conducted 030* experiments. Or something that could explain the difference between 020* and these 030* models.

File: external/mmdetection/tests/test_ote_api.py [hunk 2]
```diff
@@ -541,6 +541,18 @@ def test_training_yolox(self):
         self.end_to_end(
             osp.join('configs', 'custom-object-detection', 'cspdarknet_YOLOX'))
 
+    @e2e_pytest_api
+    def test_training_person_detection_0301(self):
+        self.end_to_end(osp.join('configs', 'person-detection', 'person-detection-0301'))
+
```
Comment (GalyaZalesskaya) [inline external/mmdetection/tests/test_ote_api.py]: I'm still not sure how we should add tests for object detection tasks. If we need to add it this way, why don't you add tests for the 020* person detectors?

File: external/mmdetection/configs/person-detection/person-detection-0301/template_experimental.yaml (hunk 1 above)
Comment (morkovka1337) [reply external/mmdetection/configs/person-detection/person-detection-0301/template_experimental.yaml]: Added more info about new models

File: external/mmdetection/tests/test_ote_api.py (hunk 2 above)
Comment (morkovka1337) [reply external/mmdetection/tests/test_ote_api.py]: Added test cases for the existing models, thx

**Comment by LeonidBeynenson** [comment]
run ote-test

**Comment by morkovka1337** [comment]
run toe-test

**Comment by morkovka1337** [comment]
run ote-test

**Comment by morkovka1337** [comment]
run toe-test

**Comment by morkovka1337** [comment]
run ote-test

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote-test

**Comment by Ilya-Krylov** [comment]
```
23:08:52            ##############################################
23:08:52            ########                              ########
23:08:52            ########  ./tests/run_code_checks.sh  ########
23:08:52            ########                              ########
23:08:52            ##############################################
23:08:52  
23:08:52  [INFO] Initializing environment for https://github.com/PyCQA/isort.
23:08:58  [INFO] Initializing environment for https://github.com/psf/black.
23:09:05  [INFO] Initializing environment for https://github.com/PyCQA/flake8.
23:09:12  [INFO] Initializing environment for https://github.com/pre-commit/mirrors-prettier.
23:09:17  [INFO] Initializing environment for https://github.com/pre-commit/mirrors-prettier:prettier@2.3.0.
23:09:22  [INFO] Initializing environment for https://github.com/pre-commit/mirrors-mypy.
23:09:29  [INFO] Initializing environment for https://github.com/pre-commit/mirrors-mypy:attrs==21.2.*,numpy==1.19.5,types-Deprecated,types-PyYAML,types-docutils,types-python-dateutil,types-requests,types_futures.
23:09:34  [INFO] Installing environment for https://github.com/PyCQA/isort.
23:09:34  [INFO] Once installed this environment will be reused.
23:09:34  [INFO] This may take a few minutes...
23:09:52  [INFO] Installing environment for https://github.com/psf/black.
23:09:52  [INFO] Once installed this environment will be reused.
23:09:52  [INFO] This may take a few minutes...
23:10:24  [INFO] Installing environment for https://github.com/PyCQA/flake8.
23:10:24  [INFO] Once installed this environment will be reused.
23:10:24  [INFO] This may take a few minutes...
23:10:34  [INFO] Installing environment for https://github.com/pre-commit/mirrors-prettier.
23:10:34  [INFO] Once installed this environment will be reused.
23:10:34  [INFO] This may take a few minutes...
23:11:00  [INFO] Installing environment for https://github.com/pre-commit/mirrors-mypy.
23:11:00  [INFO] Once installed this environment will be reused.
23:11:00  [INFO] This may take a few minutes...
23:11:19  [INFO] Installing environment for https://github.com/pre-commit/mirrors-mypy.
23:11:19  [INFO] Once installed this environment will be reused.
23:11:19  [INFO] This may take a few minutes...
23:11:31  isort (ote_sdk)..........................................................Passed
23:11:31  isort (ote_cli|external).................................................Passed
23:11:32  black (ote_sdk|ote_cli)..................................................Passed
23:11:32  black (rest).............................................................Passed
23:11:34  flake8...................................................................Passed
23:11:35  prettier.................................................................Failed
23:11:35  - hook id: prettier
23:11:35  - files were modified by this hook
23:11:35  
23:11:35  external/mmdetection/configs/person-detection/person-detection-0202/template_experimental.yaml
23:11:35  external/mmdetection/configs/person-detection/person-detection-0201/template_experimental.yaml
23:11:35  external/mmdetection/configs/person-detection/person-detection-0303/template_experimental.yaml
23:11:35  external/mmdetection/configs/person-detection/person-detection-0302/template_experimental.yaml
23:11:35  external/mmdetection/configs/person-detection/person-detection-0203/template_experimental.yaml
23:11:35  external/mmdetection/configs/person-detection/person-detection-0301/template_experimental.yaml
23:11:35  external/mmdetection/configs/person-detection/person-detection-0200/template_experimental.yaml
23:11:35  
23:11:40  mypy (ote_sdk)...........................................................Passed
23:11:41  mypy (ote_cli)...........................................................Passed
23:11:41  mypy (external)..........................................................Passed
23:12:20  pylint...................................................................Passed
23:12:20  [Pipeline] }
23:12:20  ERROR: script returned exit code 1
```

**Comment by morkovka1337** [comment]
run ote-test

**Comment by morkovka1337** [comment]
run ote_sdk tests

**Comment by morkovka1337** [comment]
run ote_sdk tests

**Comment by morkovka1337** [comment]
run ote-test

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by morkovka1337** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by morkovka1337** [comment]
run ote-test

**Comment by morkovka1337** [comment]
run ote_sdk tests

**Comment by aiarkinx** [comment]
run ote-test

**Comment by aiarkinx** [comment]
run ote-test

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by morkovka1337** [comment]
run ote-test

**Comment by morkovka1337** [comment]
run ote-test

**Comment by morkovka1337** [comment]
run ote_sdk tests

**Comment by druzhkov-paul** [comment]
Each of the last three test runs ended up having one failed test from those added in this PR, including person_detection_0200, person_detection_0203 and person_detection_0301. @morkovka1337 are you sure that tests are reliable enough for being merged to develop?

**Comment by morkovka1337** [comment]
run ote_sdk test

**Comment by morkovka1337** [comment]
run ote-test

**Comment by morkovka1337** [comment]
run ote_sdk test

**Comment by druzhkov-paul** [comment]
run ote_sdk tests

**Comment by druzhkov-paul** [comment]
run ote-test

**Pull request closed without merging**